### ISSUE TITLE
New Install - Libraries Not Populating

### ISSUE THREAD

**Issue by Phillyphotog** [ukdtom/ExportTools.bundle#160]
Running plugin version 2.0.0.15, PMS 1.21.1.3876, Plex Web 4.47.3

Tried restarting server application, followed instructions to to to Settings, Manage, Plugins, Reload Library List.

Have closed, and reopened Server app, Refreshed the webpage, tried webpage on a different machine.  Still not getting any libraries.

**Comment by Phillyphotog** [comment]
Also, not seeing logs in log location.  Pulled below out of console:
Jan 26, 2021 15:23:36.900 [10816] Debug — Request: [192.168.***.***:58498 (Subnet)] GET /applications/ExportTools/:/prefs (11 live) TLS GZIP Signed-in Token (eagle2331)
Jan 26, 2021 15:23:36.900 [10816] Debug — Plug-in com.plexapp.plugins.ExportTools has been used 28 times.
Jan 26, 2021 15:23:36.901 [10816] Debug — [com.plexapp.plugins.ExportTools] Sending command over HTTP (GET): /applications/ExportTools/:/prefs
Jan 26, 2021 15:23:36.901 [10816] Debug — HTTP requesting GET http://127.0.0.1:49379/applications/ExportTools/:/prefs
Jan 26, 2021 15:23:36.948 [10816] Debug — HTTP 200 response from GET http://127.0.0.1:49379/applications/ExportTools/:/prefs
Jan 26, 2021 15:23:36.948 [10816] Debug — [com.plexapp.plugins.ExportTools] HTTP reply status 200, with 3805 bytes of content.
Jan 26, 2021 15:23:36.950 [12468] Debug — Completed: [192.168.***.***:58498] 200 GET /applications/ExportTools/:/prefs (11 live) TLS GZIP 50ms 1600 bytes (pipelined: 1)

**Comment by Phillyphotog** [comment]
Ah, found the log for ExportTools  Have a critical error:
2021-01-26 15:48:48,296 (1e18) :  CRITICAL (core:574) - Exception when calling function 'ValidatePrefs' (most recent call last):
  File "I:\PMS\Resources\Plug-ins-3c3adfcb4\Framework.bundle\Contents\Resources\Versions\2\Python\Framework\code\sandbox.py", line 294, in call_named_function
    result = f(*args, **kwargs)
  File "I:\PMS\Resources\Plug-ins-3c3adfcb4\ExportTools.bundle\Contents\Code\__init__.py", line 518, in ValidatePrefs
    ResetToIdle()
  File "I:\PMS\Resources\Plug-ins-3c3adfcb4\ExportTools.bundle\Contents\Code\__init__.py", line 464, in ResetToIdle
    pl = plistlib.readPlist(pFile)
  File "I:\PMS\python27.zip\plistlib.py", line 75, in readPlist
    pathOrFile = open(pathOrFile)
  File "I:\PMS\Resources\Plug-ins-3c3adfcb4\Framework.bundle\Contents\Resources\Versions\2\Python\subsystem\ospathfix.py", line 70, in builtins_open
    return __builtins__['_open'](longpathify(uni(filename)), mode, buffering)
IOError: [Errno 2] No such file or directory: u'\\\\?\\I:\\Plex Media Server\\Plug-ins\\ExportTools.bundle\\Contents\\Info.plist'

**Comment by Phillyphotog** [comment]
Found issue.  Had plugin in incorrect plugin directory.  I had the plugin in the Resources  Plugin folder, in the same root directory that contains the plex executables.

Plex was able to detect and load the plugin and present the settings for the plugin.

Once I deleted it from that folder and moved it to plug-ins folder in the same root that contains media, metadata, scanners, cache, and crash reports, the plugin loaded the libraries properly per documentation. 

Knowledge gap on my side.  haven't played with plugins in a while.
